pkgs/by-name/bp/bpftrace/package.nix

```diff
@@ -52,31 +52,15 @@ stdenv.mkDerivation rec {
     util-linux
   ];
 
-  # tests aren't built, due to gtest shenanigans. see:
-  #
-  #     https://github.com/bpftrace/bpftrace/issues/161#issuecomment-453606728
-  #     https://github.com/bpftrace/bpftrace/pull/363
-  #
   cmakeFlags = [
-    "-DBUILD_TESTING=FALSE"
     "-DLIBBCC_INCLUDE_DIRS=${bcc}/include"
     "-DINSTALL_TOOL_DOCS=OFF"
     "-DSYSTEM_INCLUDE_PATHS=${glibc.dev}/include"
   ];
 
-  patches = [
-    (fetchpatch {
-      name = "runqlat-bt-no-includes.patch";
-      url = "https://github.com/bpftrace/bpftrace/pull/3262.patch";
-      hash = "sha256-9yqaZeG1Uf2cC9Aa40c2QUTQRl8n2NO1nq278hf9P4M=";
-    })
-    (fetchpatch {
-      name = "kheaders-not-found-message-only-on-error.patch";
-      url = "https://github.com/bpftrace/bpftrace/pull/3265.patch";
-      hash = "sha256-8AICMzwq5Evy9+hmZhFjccw/HmgZ9t+YIoHApjLv6Uc=";
-      excludes = [ "CHANGELOG.md" ];
-    })
-  ];
+  postPatch = ''
+    substituteInPlace CMakeLists.txt --replace "set(MAX_LLVM_MAJOR 18)" "set(MAX_LLVM_MAJOR 19)"
```

Comment:
confirmed that's all that's needed here:
https://github.com/bpftrace/bpftrace/pull/3433

so, ok with me.